tspng/test_extraction.py

```diff
@@ -77,4 +83,11 @@ def test_extract_from_folder_fails():
     #test if PNG files in directory
     with pytest.raises(Exception):
         extract_from_folder('tests')
-    
\ No newline at end of file
+
+def test_extract_from_url():
+    '''
+    Tests the dictionary keys from a list of example TSPNG url.
+    '''
+    abs_path = os.path.abspath('tests/assets/example_file_1.ts.png')
```

Comment:
This test is okay. It would be better to put a TS PNG image on Google Drive or S3 or host it some where to use the HTTP functionality. This test does not really exercise the HTTP download aspect.

Reply:
Change made as requested, used the link to the test file in the repository